docs/docsite/rst/vmware/scenario_remove_vm.rst

```diff
@@ -0,0 +1,126 @@
+.. _vmware_guest_remove_virtual_machine:
+
+**********************************
+Remove an existing virtual machine
+**********************************
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to remove an existing virtual machine.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.5 or later must be installed
+
+    * The Python module *Pyvmomi* must be installed on the Ansible (or Target host if not executing against localhost)
+
+    * Installing the latest *Pyvmomi* via pip is recommended [as the OS packages are usually out of date and incompatible]
+
+* Hardware
+
+    * At least one standalone ESXi server or
+
+    * vCenter Server with at least one ESXi server
+
+* Access / Credentials
+
+    * Ansible (or the target server) must have network access to the either vCenter server or the ESXi server
+
+    * Username and Password for vCenter or ESXi server
+
+Assumptions
+===========
+
+- All variable names and VMware object names are case sensitive.
+- You need to use Python 2.7.9 version in order to use *validate_certs* option, as this version is capable of changing the SSL verification behaviours.
+
+Caveats
+=======
+
+- Hosts in the ESXi cluster must have access to the datastore that the template resides on.
+- Module tries to mimick VMware Web UI, so virtual machine should be in powered off state in order to remove it from the VMware inventory.
```

Comment:
Which module?

Reply:
"should be removed", or "must be removed"?

Reply:
I didn't get your second comment.

Reply:
Oh, I wrote the wrong thing, I meant you've written "should be in powered off ", should that be stronger, ie "must be in powered off"

Reply:
OK, got it.